frontend/src/components/NoContent/styled.ts

```diff
@@ -1,4 +1,4 @@
-import styled from 'styled-components';
+import { styled } from '@mui/material';
 import Box from '@mui/material/Box';
```

Comment:
as in other styled files ,import can do in one line  from '@mui/material', i think

Reply:
done